crates/pop-cli/src/helpers.rs

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
Most of these 'helpers' are specific to `new parachain` and should therefore exist in the `commands::new::parachain` module instead. They are not general purpose to the whole crate and I fear this general approach is only going to result in future technical debt.

Reply:
You are right, all methods are used only in `commands::new::parachain`, so moved there